src/rulesets/__tests__/reader.jest.test.ts

```diff
@@ -353,42 +336,6 @@ describe('Rulesets reader', () => {
     ).toHaveLength(0);
   });
 
-  it('should limit the scope of formats to a ruleset', () => {
```

Comment:
The test should be kept. We just need to use different (custom) rulesets.

Reply:
@P0lip ACK

Reply:
@P0lip Tentatively replaced with `should limit the scope of formats to a ruleset`. 

Thoughts?

Reply:
@P0lip I'll rewrite this to custom rulesets later today.

Reply:
@P0lip Done. Did I manage to keep the intent of the original test?

Reply:
Yup, seems good! Thank you.